crates/astria-conductor/src/executor/tests.rs

```diff
@@ -151,6 +151,7 @@ async fn start_mock(disable_empty_block_execution: bool) -> MockEnvironment {
         &server_url,
         chain_id,
         disable_empty_block_execution,
+        1,
```

Comment:
Can you look into providing a test for skipping this block?

Reply:
Added some new tests!